### ISSUE TITLE
[BUG] <title>iOS simulator does not launch

### ISSUE THREAD

**Issue by JasonGCross** [microsoft/vscode-dotnettools#1543]
### Describe the Issue

Opening a new bug here as requested from: [Maui team](https://github.com/dotnet/maui/issues/25301)

Problem encountered on [maui tutorial](https://dotnet.microsoft.com/en-us/learn/maui/first-app-tutorial/device-setup)
Operating System: macos

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

macOS version: Sequoia 15.0.1
Visual Studio Code version: 1.94.2 (Universal)

Repro steps:

create a new blank .NET MAUI project
select iOS as a debug target
click "Run and Debug"
Expected outcome:

the iOS simulator will open
Observed outcome:

the iOS simulator never opens
BUT build does succeed, showing debugger attached
Terminal output:

Executing task: dotnet build -t:Build -p:Configuration=Debug -f net7.0-ios -r iossimulator-x64 -p:CustomAfterMicrosoftCSharpTargets="/Users/{user_name}/.vscode/extensions/ms-dotnettools.dotnet-maui-1.3.29-darwin-arm64/dist/resources/Custom.After.Microsoft.CSharp.targets" -p:MauiVSCodeBuildOutputFile="/var/folders/km/zsdqbjrd2rb6hzkwvh95r7ww0000gn/T/dotnet-maui/maui-vsc-5abdbe76-2ed9-4f95-84dd-026a618550f1.json" -p:XamlTools="/Users/{user_name}/.vscode/extensions/ms-dotnettools.csharp-2.50.27-darwin-arm64/.xamlTools" /Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj
Determining projects to restore...
All projects are up-to-date for restore.
/usr/local/share/dotnet/sdk/8.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(251,5): warning NETSDK1201: For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument. [/Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net7.0-ios]
Detected signing identity:

Bundle Id: com.companyname.mauiapp1
App Id: com.companyname.mauiapp1
Including assemblies for Hot Reload support
MauiApp1 -> /Users/{user_name}/Projects/MauiApp1/bin/Debug/net7.0-ios/iossimulator-x64/MauiApp1.dll
Optimizing assemblies for size may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink

Build succeeded.

/usr/local/share/dotnet/sdk/8.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(251,5): warning NETSDK1201: For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument. [/Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net7.0-ios]
1 Warning(s)
0 Error(s)

Time Elapsed 00:00:00.82

### Steps To Reproduce

macOS version: Sequoia 15.0.1
Visual Studio Code version: 1.94.2 (Universal)

Repro steps:

1. create a new blank .NET MAUI project
2. select iOS as a debug target
3. click "Run and Debug"

### Expected Behavior

the iOS simulator will open

### Environment Information

Observed outcome:

the iOS simulator never opens
BUT build does succeed, showing debugger attached
Terminal output:

Executing task: dotnet build -t:Build -p:Configuration=Debug -f net7.0-ios -r iossimulator-x64 -p:CustomAfterMicrosoftCSharpTargets="/Users/{user_name}/.vscode/extensions/ms-dotnettools.dotnet-maui-1.3.29-darwin-arm64/dist/resources/Custom.After.Microsoft.CSharp.targets" -p:MauiVSCodeBuildOutputFile="/var/folders/km/zsdqbjrd2rb6hzkwvh95r7ww0000gn/T/dotnet-maui/maui-vsc-5abdbe76-2ed9-4f95-84dd-026a618550f1.json" -p:XamlTools="/Users/{user_name}/.vscode/extensions/ms-dotnettools.csharp-2.50.27-darwin-arm64/.xamlTools" /Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj
Determining projects to restore...
All projects are up-to-date for restore.
/usr/local/share/dotnet/sdk/8.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(251,5): warning NETSDK1201: For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument. [/Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net7.0-ios]
Detected signing identity:

Bundle Id: com.companyname.mauiapp1
App Id: com.companyname.mauiapp1
Including assemblies for Hot Reload support
MauiApp1 -> /Users/{user_name}/Projects/MauiApp1/bin/Debug/net7.0-ios/iossimulator-x64/MauiApp1.dll
Optimizing assemblies for size may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink

Build succeeded.

/usr/local/share/dotnet/sdk/8.0.403/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(251,5): warning NETSDK1201: For projects targeting .NET 8.0 and higher, specifying a RuntimeIdentifier will no longer produce a self contained app by default. To continue building self-contained apps, set the SelfContained property to true or use the --self-contained argument. [/Users/{user_name}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net7.0-ios]
1 Warning(s)
0 Error(s)

Time Elapsed 00:00:00.82

Terminal will be reused by tasks, press any key to close it

**Comment by emaf** [comment]
Does this happen if you target .NET 8?

Also, are you running on an Intel based Mac?

**Comment by JasonGCross** [comment]
Hi,
My Mac is an Apple Silicon Mac.

**Comment by JasonGCross** [comment]
Hi again, after changing the project settings from .NET 7 to .NET 8, there is a new error:

 Restored /Users/{username}/Projects/MauiApp1/MauiApp1.csproj (in 296 ms).
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk.net8.0_18.0/18.0.8303/tools/msbuild/iOS/Xamarin.Shared.targets(1835,3): error : Could not find any available provisioning profiles for MauiApp1 on iOS. [/Users/{username}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net8.0-ios]
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk.net8.0_18.0/18.0.8303/tools/msbuild/iOS/Xamarin.Shared.targets(1835,3): error :          [/Users/{username}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net8.0-ios]

Build FAILED.

/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk.net8.0_18.0/18.0.8303/tools/msbuild/iOS/Xamarin.Shared.targets(1835,3): error : Could not find any available provisioning profiles for MauiApp1 on iOS. [/Users/{username}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net8.0-ios]
/usr/local/share/dotnet/packs/Microsoft.iOS.Sdk.net8.0_18.0/18.0.8303/tools/msbuild/iOS/Xamarin.Shared.targets(1835,3): error :          [/Users/{username}/Projects/MauiApp1/MauiApp1.csproj::TargetFramework=net8.0-ios]
    0 Warning(s)
    1 Error(s)


TLDR: 
> Could not find any available provisioning profiles for MauiApp1 on iOS. 

Question: can one set a provisioning profile in VS Code? The official Microsoft Docs only mention Visual Studio, which is now deprecated: [https://learn.microsoft.com/en-us/dotnet/maui/ios/device-provisioning/manual-provisioning?view=net-maui-8.0](https://learn.microsoft.com/en-us/dotnet/maui/ios/device-provisioning/manual-provisioning?view=net-maui-8.0)

**Comment by JasonGCross** [comment]
Hi,

copying @jstedfast 

It is possible that a physical device was selected instead of a simulator. I am no longer getting a provisioning warning. 

But the original issue is still present -- cannot launch simulator.

Note: if I use a different (real) IDE -- JetBrains Rider -- it will launch the iOS simulator. But not VS Code.

Here is the terminal output from my latest run:

 *  Executing task: dotnet build -t:Build -p:Configuration=Debug -f net8.0-maccatalyst -r maccatalyst-arm64 -p:CustomAfterMicrosoftCSharpTargets="/Users/jasoncross/.vscode/extensions/ms-dotnettools.dotnet-maui-1.4.36-darwin-arm64/dist/resources/Custom.After.Microsoft.CSharp.targets" -p:MauiVSCodeBuildOutputFile="/var/folders/km/zsdqbjrd2rb6hzkwvh95r7ww0000gn/T/dotnet-maui/maui-vsc-5767ea2a-a3ee-4137-ab93-e8f0c76e21b8.json" -p:XamlTools="/Users/jasoncross/.vscode/extensions/ms-dotnettools.csharp-2.50.27-darwin-arm64/.xamlTools" /Users/jasoncross/Projects/MauiApp3/MauiApp3.csproj 

  Determining projects to restore...
  Restored /Users/jasoncross/Projects/MauiApp3/MauiApp3.csproj (in 345 ms).
  Detected signing identity:
          
    Code Signing Key: "" (-)
    Bundle Id: com.companyname.mauiapp3
    App Id: com.companyname.mauiapp3
  Including assemblies for Hot Reload support
  MauiApp3 -> /Users/jasoncross/Projects/MauiApp3/bin/Debug/net8.0-maccatalyst/maccatalyst-arm64/MauiApp3.dll
  Optimizing assemblies for size may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink
  Optimizing assemblies for size. This process might take a while.

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:17.39
 *  Terminal will be reused by tasks, press any key to close it. 

 *  Executing task: dotnet build -t:Build -p:Configuration=Debug -f net8.0-ios -r iossimulator-arm64 -p:CustomAfterMicrosoftCSharpTargets="/Users/jasoncross/.vscode/extensions/ms-dotnettools.dotnet-maui-1.4.36-darwin-arm64/dist/resources/Custom.After.Microsoft.CSharp.targets" -p:MauiVSCodeBuildOutputFile="/var/folders/km/zsdqbjrd2rb6hzkwvh95r7ww0000gn/T/dotnet-maui/maui-vsc-68833743-4657-4299-a14b-7c7cfe7e8fb9.json" -p:XamlTools="/Users/jasoncross/.vscode/extensions/ms-dotnettools.csharp-2.50.27-darwin-arm64/.xamlTools" /Users/jasoncross/Projects/MauiApp3/MauiApp3.csproj 

  Determining projects to restore...
  Restored /Users/jasoncross/Projects/MauiApp3/MauiApp3.csproj (in 302 ms).
  Detected signing identity:
          
    Bundle Id: com.companyname.mauiapp3
    App Id: com.companyname.mauiapp3
  Including assemblies for Hot Reload support
  MauiApp3 -> /Users/jasoncross/Projects/MauiApp3/bin/Debug/net8.0-ios/iossimulator-arm64/MauiApp3.dll
  Optimizing assemblies for size may change the behavior of the app. Be sure to test after publishing. See: https://aka.ms/dotnet-illink
  Optimizing assemblies for size. This process might take a while.

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:16.40
 *  Terminal will be reused by tasks, press any key to close it.
